testing/correctness/playbooks/boundary-reconnect.md

```diff
@@ -24,12 +24,12 @@ The gist of the test is simple: We run `sequence_window` with `Spike` on the `in
 
 ### Setting Up for the Test:
 
-1. build Giles receiver and sender
+1. build Data Receiver and Giles Sender
 2. build `testing/correctness/apps/sequence_window` with `-D spike` (and optionally `-d` for debug messages)
 
 ### Running the Test:
 
-1. start giles-receiver:  `../../../../giles/receiver/receiver --ponythreads=1 --ponynoblock --ponypinasio -l 127.0.0.1:5555`
+1. start giles-receiver:  `../../../../utils/data_receiver/data_receiver --framed --ponythreads=1 --ponynoblock --ponypinasio -l 127.0.0.1:5555`
```

Comment:
- stlll says "start giles-receiver"
- need to pipe output to file: ` > received.txt`